src/mss/linux.py

```diff
@@ -194,6 +202,25 @@ class XWindowAttributes(Structure):
 _XRANDR = find_library("Xrandr")
 
 
+class XError(ScreenShotError):
+    def __str__(self):
+        msg = super().__str__()
+        # We use something similar to the default Xlib error handler's format, since that's quite
+        # well-understood.  The original code is in
+        # https://gitlab.freedesktop.org/xorg/lib/libx11/-/blob/master/src/XlibInt.c?ref_type=heads#L1313
+        # but we don't try to implement most of it.
+        msg += (
+            f"\nX Error of failed request:  {self.details["error"]}"
+            f"\n  Major opcode of failed request:  {self.details["request_code"]} ({self.details["request"]})")
```

Comment:
```suggestion
            f"\n  Major opcode of failed request:  {self.details['request_code']} ({self.details['request']})")
```

+ same for other lines should fix the syntax error.